src/SDKs/ServiceBus/data-plane/tests/Microsoft.Azure.ServiceBus.Tests/RetryTests.cs

```diff
@@ -130,7 +130,8 @@ async void RunOperationShouldReturnImmediatelyIfRetryIntervalIsGreaterThanOperat
             var policy = RetryPolicy.Default;
             var watch = Stopwatch.StartNew();
             await Assert.ThrowsAsync<ServiceBusException>(async () => await policy.RunOperation(
-                    () => throw new ServiceBusException(true, string.Empty), TimeSpan.FromSeconds(8)));
+                    () => throw new ServiceBusException(true, string.Empty), TimeSpan.FromSeconds(8)))
+                .ConfigureAwait(false);
```

Comment:
I had removed this in a previous PR.   Despite it seeming to be an innocuous change, restoring here as there are quite a few areas of the tests that rely on not restoring the context.   I'm open to suggestion, should anyone feel that it's better left gone.

Reply:
> few areas of the tests that rely on not restoring the context

What relies on that?

Reply:
Since a single test often serves as a producer and consumer of messages, they've got the message sender working in the background and typically the receiver working on the original context.   I made the mistake of removing some before I understood what the intent was and things blew up rather spectacularly.

Reply:
Is the receiver blocking? If it's async there should be no problem for subsequent continuations to get back on the original context.

Reply:
They're interleaving delays, multiple senders, multiple receivers for different scenarios.   Intuitively, I expected it to be safe to remove, but the reality was very different.  I honestly didn't see justification to spend the time to really dig into the details of why they went with the pattern that they did.  You're welcome to dig deeper and file issues to refactor things, but I'm going to call it out of scope for now.

Reply:
Sounds good.